styleguide.config.js

```diff
@@ -203,6 +203,10 @@ module.exports = {
             name: "Spacing",
             content: "packages/wonder-blocks-spacing/docs.md",
         },
+        {
+            name: "Testing",
+            content: "packages/wonder-blocks-testing/docs.md",
+        },
```

Comment:
Probably worth mentioning that we won't support Styleguide in the short term. The plan is to migrate to Storybook docs in January/February, so probably it would be better if you can add the story for that instead 😃. That would reduce the effort to migrating later to Storybook, and that'd be highly appreciated 🙏.

Reply:
I don't intend on updating the styleguidist doc beyond this one file. I'll add stories when there's something to add stories for :D

Reply:
sounds good, thanks for the clarification!